### PR TITLE
Improve kids page seo

### PR DESCRIPTION
**Pull request by baryska**

### CODE REVIEW

File: pages/kids/index.tsx [hunk 1]
```diff
@@ -15,8 +15,10 @@ const KidsSection = () => {
       <Head>
         <meta name="referrer" content="no-referrer" />
         <title>{t('seo.dictionary_page_title')}</title>
```
Comment (met) [inline pages/kids/index.tsx]: Changing also <title> tag to seo.kids_page_title

**Comment by met** [comment]
✔ SEO texts works good after small fix. 
❌ But not social network preview, it still show the old (common) image.

Testing for Facebook here: https://developers.facebook.com/tools/debug/
Testing for Twitter here: https://cards-dev.twitter.com/validator

Problem is in the file `pages/_document.tsx` that contains global `meta` tags with pictures for Facebook and Twitter.

https://github.com/cesko-digital/movapp/blob/536b25aefa53addd6dd0560d6de4f49aaf54a7c1/pages/_document.tsx#L16
https://github.com/cesko-digital/movapp/blob/536b25aefa53addd6dd0560d6de4f49aaf54a7c1/pages/_document.tsx#L20

Rendered Kids page contains two sets of metatags one from _document.tsx and another from kids pages, which causes the problem. It looks that Facebook and Twitter prefere the second tags:

image

I checked the [documentation, where is written](https://nextjs.org/docs/advanced-features/custom-document): 

> The <Head /> component used in _document is not the same as [next/head](https://nextjs.org/docs/api-reference/next/head). The <Head /> component used here should only be used for any <head> code that is common for all pages. For all other cases, such as <title> tags, we recommend using [next/head](https://nextjs.org/docs/api-reference/next/head) in your pages or components.

It means we cannot overrited what is set in _document we can only duplicate it.

I am refusing the PR for now. But if you could not find an easy solution then please ask again and I will accept it. It is still an improvement and nothing is broken with this. Just the preview picture does not work. 😉

**Review by met — DISMISSED** [review]

**Comment by baryska** [comment]
I've created a new component with all SEO tags instead of Head for each page separately. However, the preview for FB and Twitter on their deveoplers page doesn't work, since the picture is not deployed on movapp.cz page yet, I suppose :) Could someone who understands metatags check whether the tags in SEO componet are sufficient (@petrillek ?) or we need to add something else? And can someone who knows NextJS better than me (that's practically everyone :D) can check whether this solution is ok? (@JuroUhlar ?) Thanks!

**Comment by met** [comment]
Perfect. You did big improvement and cleaning of META tags.  👍
I see all tags, that should be necessary.

Only this one I do not understand  `<meta name="name" content={title} />`
I believe, this is not necesary (even if such meta really exists).

image

**Comment by baryska** [comment]
@JuroUhlar Mohl by ses prosím podívat, jestli je tohle řešení v rámci NextJS ok? Díky!

**Review by JuroUhlar — APPROVED** [review]
Good job! I think it works well. You can remove the `<meta name="name"` as Martin suggested and then merge the changes.

**Comment by JuroUhlar** [comment]
@baryska  One more thing: Your SEO component is a normal component, it can have a descriptive file name. Only page components need to be an index.tsx file inside a separate folder (Next has file-based routes, folder inside pages = route).

**Pull request merged**